apps/avifdec.c

```diff
@@ -47,6 +47,7 @@ static void syntax(void)
     printf("                        Default: %u, set to a smaller value to further restrict.\n", AVIF_DEFAULT_IMAGE_SIZE_LIMIT);
     printf("  --dimension-limit C : Specifies the image dimension limit (width or height) that should be tolerated.\n");
     printf("                        Default: %u, set to 0 to ignore.\n", AVIF_DEFAULT_IMAGE_DIMENSION_LIMIT);
+    printf("    --                : Signals the end of options. Everything after is interpreted as file names.\n");
```

Comment:
Optional: add "this" before "is"

If you make this change, please make the same change to the other occurrences. (Search for "Everything after is".)